regression-test/suites/cross_ds/fullsync/insert_overwrite/test_ds_dml_insert_overwrite_fs.groovy

```diff
@@ -0,0 +1,149 @@
+    INSERT INTO ${uniqueTable} VALUES
+        (1, 0),
+        (1, 1),
+        (1, 2),
+        (1, 3),
+        (1, 4)
+    """
+    sql "sync"
+
+    // test 1: target cluster follow source cluster
+    logger.info("=== Test 1: backup/restore case ===")
+
+    helper.ccrJobDelete()
+    helper.ccrJobCreate()
+    assertTrue(helper.checkRestoreFinishTimesOf("${uniqueTable}", 60))
+    assertTrue(helper.checkShowTimesOf("SELECT * FROM ${uniqueTable} WHERE test = 1", exist, 60, "sql"))
+    assertTrue(helper.checkShowTimesOf("SELECT * FROM ${uniqueTable} WHERE test = 1", exist, 60, "target"))
+    assertTrue(helper.checkSelectTimesOf("SELECT * FROM ${uniqueTable} WHERE test = 1 ORDER BY id", 5, 60))
+
+    logger.info("=== Test 2: dest cluster follow source cluster case ===")
+
+    sql """
+    INSERT INTO ${uniqueTable} VALUES
+        (2, 0),
+        (2, 1),
+        (2, 2),
+        (2, 3),
+        (2, 4)
+    """
+    sql "sync"
+
+    assertTrue(helper.checkShowTimesOf("SELECT * FROM ${uniqueTable} WHERE test=2", exist, 60, "sql"))
+    assertTrue(helper.checkShowTimesOf("SELECT * FROM ${uniqueTable} WHERE test=2", exist, 60, "target"))
+    assertTrue(helper.checkSelectTimesOf("SELECT * FROM ${uniqueTable} WHERE test=2", 5, 60))
+
+    logger.info("=== Test 3: insert overwrite source table ===")
+
+    sql """
+    INSERT OVERWRITE TABLE ${uniqueTable} VALUES
+        (3, 0),
+        (3, 1),
+        (3, 2),
+        (3, 3),
+        (3, 4)
+    """
+    sql "sync"
+
+    assertTrue(helper.checkShowTimesOf("SELECT * FROM ${uniqueTable} WHERE test=3", exist, 60, "sql"))
+    assertTrue(helper.checkShowTimesOf("SELECT * FROM ${uniqueTable} WHERE test=2", notExist, 60, "sql"))
+    assertTrue(helper.checkShowTimesOf("SELECT * FROM ${uniqueTable} WHERE test=3", exist, 60, "target"))
+    assertTrue(helper.checkShowTimesOf("SELECT * FROM ${uniqueTable} WHERE test=2", notExist, 60, "target"))
+
+    assertTrue(helper.checkSelectTimesOf("SELECT * FROM ${uniqueTable} WHERE test=3", 5, 60))
+    assertTrue(helper.checkSelectTimesOf("SELECT * FROM ${uniqueTable}", 5, 60))
+	
+	logger.info("=== Test 4: pause job --> insert overwrite --> force full sync --> resume")
+	helper.ccrJobPause()
+	
+	target_sql """
+        INSERT OVERWRITE TABLE ${uniqueTable} VALUES
+		(5, 0)
+        """
+	target_sql "sync"
+	
+	assertTrue(helper.checkShowTimesOf("SELECT * FROM ${uniqueTable} WHERE test=5", exist, 60, "target"))
+	
+	sql """
+    INSERT OVERWRITE TABLE ${uniqueTable} VALUES
+        (4, 0),
+        (4, 1),
+        (4, 2),
+        (4, 3),
+        (4, 4)
+    """
+    sql "sync"
+	
+	helper.force_fullsync()
+	helper.ccrJobResume()
+	
+	assertTrue(helper.checkShowTimesOf("SELECT * FROM ${uniqueTable} WHERE test=4", exist, 60, "sql"))
+	assertTrue(helper.checkShowTimesOf("SELECT * FROM ${uniqueTable} WHERE test=4", exist, 60, "target"))
+	assertTrue(helper.checkShowTimesOf("SELECT * FROM ${uniqueTable} WHERE test=5", notExist, 60, "target"))
```

Comment:
Consider moving these lines into `regression-test/suites/db_sync/dml/insert_overwrite/test_ds_dml_insert_overwrite.groovy`, since they have the same aims.